include/tvm/tir/transform.h

```diff
@@ -484,6 +484,13 @@ TVM_DLL Pass MergeDynamicSharedMemoryAllocations();
  */
 TVM_DLL Pass ConvertForLoopsToSerial();
 
+/*!
+ * \brief This is the unified static memory planner pass that will
+ * plan for memory intra- and inter- PrimFuncs together.
+ * \return The pass.
```

Comment:
nit: state reqs all globals to be PrimFuncs, inc main.

Reply:
Done